File: Java/waiting_for_spring.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;
import java.util.Scanner;

public class game_logs {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String city_name = "";
        String datas = "";
        String[] line_dat;
        ArrayList<Integer> tempData = new ArrayList<>();
        int countWarmMonths = 0;

        System.out.print("cityName: ");
        city_name = in.nextLine();

        System.out.print("tempData: ");
        datas = in.nextLine();
        line_dat = datas.split(",");

        for(int i = 0; i < 12; i++){
            int temp = Integer.parseInt(line_dat[i]);
            if (temp >= 20){
                countWarmMonths++;
            }
        }

        System.out.println(city_name + " has " + countWarmMonths + " warm months.");

    }

}

File: Python/waiting_for_spring.py
def solution(city, temp_data):
    count = 0
    for i in temp_data:
        if i >= 20:
            count += 1
    return f"{city} has {count} warm months."


if __name__ == '__main__':
    test = input()
    city = input("CityName: ")
    temp_data = list(map(int, input("TempData: ").strip().split(',')))[:12]
    print(solution(city, temp_data))